packages/flutter/lib/src/material/range_slider.dart

```diff
@@ -454,25 +454,25 @@ class _RangeSliderState extends State<RangeSlider> with TickerProviderStateMixin
   }
 
   // Finds closest thumb. If the thumbs are close to each other, no thumb is
-  // immediately selected while the drag velocity is zero. If the first
-  // non-zero velocity is negative, then the left thumb is selected, and if its
+  // immediately selected while the drag displacement is zero. If the first
+  // non-zero displacement is negative, then the left thumb is selected, and if its
   // positive, then the right thumb is selected.
   static final RangeThumbSelector _defaultRangeThumbSelector = (
       TextDirection textDirection,
       RangeValues values,
       double tapValue,
       Size thumbSize,
       Size trackSize,
-      double dx, // drag velocity
+      double dx, // drag displacement
     ) {
     final double touchRadius = math.max(thumbSize.width, RangeSlider._minTouchTargetWidth) / 2;
     final bool inStartTouchTarget = (tapValue - values.start).abs() * trackSize.width < touchRadius;
     final bool inEndTouchTarget = (tapValue - values.end).abs() * trackSize.width < touchRadius;
 
-    // Use the velocity if the thumb touch targets overlap. If dx is 0 and the
-    // the drag position is in both touch targets, no thumb is selected because
-    // it is ambiguous to which thumb should be selected. Once the drag
-    // velocity is non-zero, the thumb selection can be determined.
+    // Use the displacement if the thumb touch targets overlap. If dx is 0 and
+    // the the drag position is in both touch targets, no thumb is selected
+    // because it is ambiguous to which thumb should be selected. Once the drag
+    // displacement is non-zero, the thumb selection can be determined.
```

Comment:
Instead of saying "can be determined", briefly explain what the thumb selection algorithm is supposed to be.

Reply:
Done